pkgs/games/frogatto/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildEnv, stdenv, callPackage, makeWrapper, makeDesktopItem }:
+
+let
+  description = "Action-adventure game, starring a certain quixotic frog";
+  engine = callPackage ./engine.nix { };
+  data = callPackage ./data.nix { };
```

Comment:
> https://github.com/NixOS/nixpkgs/pull/52989#issuecomment-450760496

These attributes are only accessible in this derivation.

You'd need move these to all-packages.nix like

```
anura-engine = callPackage ../games/frogatto/engine.nix { }; # remember alphabetical

 
frogatto-data = callPackage ../games/frogatto/data.nix { };
```
And then use that attribute in here.

Reply:
Should be ok, as frogatto seems to be its only user.

Reply:
The engine in its current form would not be useful for other consumers of the engine anyway.